### ISSUE TITLE
Feature: Add background jobs to docker

### ISSUE THREAD

**Issue by steschuser** [ngoduykhanh/PowerDNS-Admin#939]
PowerDNS-Admin supports the fetching of Domain and Account data in the background.
This results in a fast and snappy website.

For this to work, the setting "bg_domain_updates" has to be turned on in Settings-> Basic and the actual fetching of the data has to happen elsewhere.

One way is to request the route /domains-updater via cronjob or similar.
This needs a login and so exposes the password/token. It can also become a hassle if you have multiple instances running.

Alternative to that are the provided scripts update_zone and update_accounts. Those work fine and dont need additional settings as they can work with the PowerDNS-Admin database directly. This makes using the scripts superior to calling /domains-updater, imho.

However, this is impossible to setup with the provided Docker Container.
Therefor another Dockerfile is needed that enables the User to switch to Background jobs easily, without setting up external cronjobs.

TLDR:
Make "bg_domain_updates" work in Docker

**Comment by steschuser** [comment]
#940
